swaggen/src/test/java/mock/endpoint/MockPostEndpoint.java

```diff
@@ -24,7 +26,11 @@
                 "param4 = Another String Param"
                 },
         body="body.scheme",
-        responses={"200=OK", "400=Bad Request", "404"},
+        responses={
+        		@SwaggerResponse(body=@SwaggerBody(value="/test/schema.json")), 
```

Comment:
This isn't an issue, just letting you know to save you some time in the future: the `value` property of an annotation doesn't need to be explicitly specified when it is the only parameter. You can just write `@SwaggerBody("/test/schema.json")`.